CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 ### 0.7.1 (Next)
 
 * [#87](https://github.com/codegram/hyperclient/pull/87): Fix: eager delegation causes link skipping - [@dblock](https://github.com/dblock).
+* [#89](https://github.com/codegram/hyperclient/issues/89): Add method fetch for Hyperclient:: Resource - [@alabeduarte](https://github.com/alabeduarte)
```

Comment:
Nitpicks ;)

There's a space after `::`.
Missing a period at the end.

fix via `--amend` please.